base_dockerfile/Dockerfile

```
@@ -0,0 +1,5 @@
+FROM centos
+ARG experiment_files
+COPY ${experiment_files}* ./
```

Refers to the previous suggestion.
```suggestion
COPY ${experiment_files}/* ./
```